README.md

```diff
@@ -108,22 +108,26 @@ $ make && ctest
 
 Celeritas guarantees full compatibility and correctness only on the
 combinations of compilers and dependencies tested under continuous integration:
-- Compilers:
-    - GCC 8.4, 12.3
-    - Clang 10.0, 15.0
-    - GCC 11.3 + NVCC 11.8
-    - HIP-Clang 15.0
+- Compilers and standard:
+    - GCC 8, 11, 12, 14
+    - Clang 10, 15, 18
+    - GCC 11.5 + NVCC 12.6
+    - ROCm Clang 18
+    - C++17 and C++20
```

Comment:
The standard coverage might depend on the version of Geant4, is it worth mentioning here?

Reply:
All these apply to the 11.0.4 version being tested, which is compatible with both 17 and 20.